frontend/lib/store/enrollment-store.ts

```diff
@@ -76,9 +76,9 @@ const fetchLaunchContext = async (set: (partial: StoreState | Partial<StoreState
     } else {
         //TODO: We can remove this when going live, this is useful during development
         launchContext = {
-            "patient": "Patient/2",
-            "serviceRequest": "ServiceRequest/4",
-            "practitioner": "Practitioner/7"
+            "patient": "Patient/4",
+            "serviceRequest": "ServiceRequest/6",
+            "practitioner": "Practitioner/8"
```

Comment:
I'd like to remove this sooner, rather than later. Why do we need this?

Reply:
It's mainly so I can test my changes in a few seconds ([Fast Refresh](https://nextjs.org/docs/architecture/fast-refresh)) on the dev server. These values map onto the first created records when using the dev server. The front-end can be started next to it, in a dev server, and just use that data. Otherwise you need to keep building docker images and test it via the dev deployment.

Reply:
I'll approve if you make an issue for it on the backlog

Reply:
https://linear.app/zorg-bij-jou/issue/INT-471/orca-or-remove-the-dev-server-launch-context